age.md

```diff
@@ -0,0 +1,296 @@
+# age
+
+[c2sp.org/age](https://c2sp.org/age),
+[age-encryption.org/v1](https://age-encryption.org/v1)
```

Comment:
How do we want maintainer and version information to be represented in specs? Or should that be in an external index? (I prefer the former.)